### PR TITLE
Remove support for Mobile Websites

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
there's DEFAULT_WEBSITE_PROJECT_TEMPLATE in our config.json's, shouldn't we delete it ?

File: lib/dynamic-help-provider.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ export class DynamicHelpProvider implements IDynamicHelpProvider {
 		return ((): IDictionary<any> => {
 			let isHtml = options.isHtml;
 			let localVariables:IDictionary<any> = {};
-			localVariables["isMobileWebsite"] = isHtml || this.isProjectType([this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite]).wait();
```
Comment (rosen-vladimirov) [inline lib/dynamic-help-provider.ts]: there are a lot of
```JavaScript
<% if (isMobileWebsite) ...
```
or some similar ones, which use this variable in our .md files. We should remove the usage from there or keep this variable.

Comment (teobugslayer) [inline lib/dynamic-help-provider.ts]: Damn, I have cleaned up these once but then rolled back this change.

**Comment by rosen-vladimirov** [comment]
`mobileWebsiteProject` is still registered in our bootstrap.ts
https://github.com/Icenium/icenium-cli/blob/totev/remove-websites/lib/bootstrap.ts#L90

**Comment by rosen-vladimirov** [comment]
We shouldn't need this file anymore:
https://github.com/Icenium/icenium-cli/blob/totev/remove-websites/resources/project-properties-mobilewebsite.json

**Comment by rosen-vladimirov** [comment]
We shouldn't need this file anymore:
https://github.com/Icenium/icenium-cli/blob/totev/remove-websites/test/resources/blank-MobileWebsite.abproject

**Comment by rosen-vladimirov** [comment]
In README.md Quick Start we have:
```
You can set hybrid, native or website for <Type>.
```
https://github.com/Icenium/icenium-cli/blob/totev/remove-websites/README.md#quick-start

**Comment by rosen-vladimirov** [comment]
Also in point of README.md's QuickStart - **3. Run in simulator** there's:

```
This operation is applicable to hybrid and mobile website projects.
```
https://github.com/Icenium/icenium-cli/blob/totev/remove-websites/README.md#quick-start

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**